plugin/EstateList.php

```diff
@@ -674,14 +674,7 @@ public function formatParkingLot(array $parkingArray, string $language, string $
 			if (!$parking['Count']) {
 				continue;
 			}
-			switch ($language) {
-				case 'ENG':
-					$element = sprintf("%d %s at %s", number_format_i18n($parking['Count']), $this->getParkingName($key, $parking['Count']), $this->formatPrice($parking['Price'], $language, $locale));
-					break;
-				default:
-					$element = sprintf("%d %s à %s", number_format_i18n($parking['Count']), $this->getParkingName($key, $parking['Count']), $this->formatPrice($parking['Price'], $language, $locale));
-					break;
-			}
+			$element = sprintf("%s at %s", $this->getParkingName($key, $parking['Count']), $this->formatPrice($parking['Price'], $language, $locale));
```

Comment:
`%1$d` with ongoing numbers should be used here instead of `%s` so that both kinds of phrases become possible:
* _$500 per garage, 5 available_ (I just noticed this is not possible right now, because the number gets inserted at a specific position in the switch/case below)
* _5 garages, $500 each_

The format needs to be wrapped in the `__()` function as well.